utility/src/main/java/org/oppia/android/util/system/BUILD.bazel

```
@@ -0,0 +1,50 @@
+"""
+Utilities corresponding to simplify using Oppia Clock.
```

```suggestion
Utilities corresponding to system utilities such as the system clock.
```

This may be a bit clearer way to describe the package.